### PR TITLE
Initial mapi run command setup

### PR DESCRIPTION
**Pull request by james-kessler**
I found lots of useful bits in the github action that make sense over here as well.

### CODE REVIEW

File: src/commands/scan.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    description: Scan duration
+    default: auto
+  # TODO: sarif + html
+  # sarif-report:
+  #   type: string
+  #   description: SARIF output path. You can upload it to GitHub using the 'github/codeql-action/upload-sarif@v1' action
+  # html-report:
+  #   type: string
+  #   description: HTML output path. You can artifact to GitHub using the 'actions/upload-artifact@v2' action
+  experimental-rewrite-plugin:
```
Comment (swarkentin) [inline src/commands/scan.yml]: Can probably drop the `experimental-` prefix from this and the below option. I will add a separate ticket to do the same for the `mapi` cli.

File: src/scripts/scan.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+docker run -it \
+  --env MAPI_TOKEN \
+  --env GITHUB_TOKEN \
+  --env POSTMAN_ENVIRONMENT \
+  --env POSTMAN_API_KEY \
+  forallsecure/mapi:latest \
```
Comment (swarkentin) [inline src/scripts/scan.sh]: Do we need to support APIs that are running locally in a separate docker network. ie: using the `--network` flag?

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:66a3bcd66133ffc5507f69240ac7950f1ee9663b

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:ed43d8c7a7d7d41b6995e027bd2fda09c1155777

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:5a30d58578a3beb6c97845ca7ee0679eeb3e09fa

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:25399c74b3c27f18674e52028e5435ceb2600953

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:a2afafd0eb5f29a67e3388019c871b8ad402f754

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:9d84f6a60dbf08b64d3807326903fa2a84a3a0d5

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:f68a22210ae59f93332561c30a8cc7115be02642

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:e22bd6ddadd0003cd3619613f6ae464a7e6f4121

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:18b24b7edd1c3f835bfc0afd418df874e2ceba5a

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:8475f83ddf051ebfd75bc5f92bd59532cf89c936

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:3941e05b4c25abc1d0930d8613e6a438525146cc

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:5927bc482a6aaa9a3d255714a914db2f14547547

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:13e13e6417e82082f90a48ef7c21fa18456aea7d

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:6cee0b3541787e0febbc66cdc85e13cf6f703102

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:5773d04a676b3e60e4177eae522802950d21e229

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:c1e9278813472908afbda046fbb6edeaac9321b6

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:0024cdde7fdb34c54b60d58e8b059d549bd79714

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:217f579b5a958cf8ff896d1bbc5715ef922be894

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:7a39e1d58dce20c7146b37adf8f42ba590003692

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:d396cd8bc17ea2432a956f1ddd968467f4b72e60

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:38b7d273cee44092fdb40369ed8f21b22e2110b7

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:408b54664a43f173c93ed78145e04e62f844473a

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:2b3a69edcd163224f5b9049350e537ab67f632a1

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:f696140d368461f250aab150dd2d7a7b12cbba52

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:7e96ac558acf5213cf01512d5907347c4141f001

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:22cc947943ebb399abeb4de43c47b4b18d6aa262

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:f304a30b43303baffcc44d3177205eb339012daf

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:312358b8a9d0c2f4476c73af0e7b3ea52b36c3a5

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:7bb3f1424ef23de3a748a62fa2fccfe13b608ceb

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:0569d072aceb07e30cde6baca231706f7e283840

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:1fbfe892da5df6542e9d92edd8352a16c0feeb85

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:57c87032de75563bb99a0fab66bc20d82307415e

File: src/commands/scan.yml (hunk 1 above)
Comment (james-kessler) [reply src/commands/scan.yml]: I will revisit this after the CLI has been updated!

**Review by swarkentin — APPROVED** [review]

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:dd8c012478fdb5bd317d46bda9766eb546453263

**Pull request merged**